### PR TITLE
Add a11y addon

### PR DESCRIPTION
**Pull request by easrng**
**Changes**

Add a11y addon that adds a category to the toolbox with blocks to allow people to add WAI-ARIA roles and labels to their sprites.
Two scratch blocks. The first one has the text "set sprite aria role to" then an input with the text "button" in it. The second one has the text "set sprite label to" then an input with the text "click me" in it.

**Reason for changes**

Scratch projects aren't accessible to people using assistive technologies. I think that is an issue.

**Tests**

I have tested it and fixed all bugs I've found.

### CODE REVIEW

**Comment by pufferfish101007** [comment]
The schema validation doesn't like the empty info array in the addon.jsom - you should delete that.

File: addons/a11y/addon.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "style.css",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "tags": ["beta"],
```
Comment (pufferfish101007) [inline addons/a11y/addon.json:22]: This should include 'editor' as well.

**Review by pufferfish101007 — CHANGES_REQUESTED** [review]

**Comment by apple502j** [comment]
This is rejected, as this essentially adds new blocks.

**Comment by WorldLanguages** [comment]
Reopening for now, might need discussion

**Comment by jeffalo** [comment]
the custom block colour addon should show this

**Pull request merged**